modules/appeals_api/spec/models/notice_of_disagreement_spec.rb

```diff
@@ -178,7 +180,50 @@
 
       notice_of_disagreement.update_status!(status: 'pending')
 
-      expect(handler).to have_received(:handle!)
+      expect(handler).to have_received(:handle!).exactly(1).times
+    end
+
+    it 'sends an email when emailAddressText is present' do
+      handler = instance_double(AppealsApi::Events::Handler)
+      allow(AppealsApi::Events::Handler).to receive(:new).and_return(handler)
+      allow(handler).to receive(:handle!)
+
+      notice_of_disagreement.update_status!(status: 'submitted')
+
+      expect(handler).to have_received(:handle!).exactly(2).times
+    end
+
+    it 'successfully gets the ICN when email isn\'t present' do
+      notice_of_disagreement = described_class.create!(
+        auth_headers: auth_headers,
+        form_data: form_data.deep_merge({
+                                          'data' => {
+                                            'attributes' => {
+                                              'veteran' => {
+                                                'emailAddressText' => nil
```

Comment:
We'll need a test for NOD v2 as well, at some point, since we renamed this key to `email` there.